docs/articles/2023/newsletter_september2023.md

```diff
@@ -0,0 +1,123 @@
+# Introducing Schema V1.1.0: Key Updates in Census API and Data Layer
+
+*Published: 25 August 2023*  
+*By: [Author Name](mailto:author1@chanzuckerberg.com), [Co-Author](mailto:author2@chanzuckerberg.com)*
+
+The Census team is thrilled to announce the introduction of Census schema V1.1.0. This version offers enhanced metadata and API functionalities, tailored to empower your single-cell research.
+
+These features are currently experimental and exclusive to the "latest" version of Census. We invite your feedback as you explore these novel functionalities.
+
+## Schema Changes in V1.1.0
+
+### Added `dataset_version_id` to Census Table
+
+The new field `dataset_version_id` has been introduced in `census_obj["census_info"]["datasets"]` to facilitate dataset versioning and management.
+
+### Added New Library-Size Normalized Layer in X["normalized"]
+
+We've introduced a library-size normalized layer in X["normalized"]. A small sigma value has been consistently added to each position, ensuring that no explicit zeros are recorded.
+
+### Enhanced Metadata Fields in `ms["RNA"].var` DataFrame
+
+The `ms["RNA"].var` DataFrame has been enriched with two new metadata fields: `nnz` and `n_measured_obs`, which provide a count of non-zero values and "measured" cells, respectively.
+
+### Enhanced Metadata Fields in `ms["RNA"].obs` DataFrame
+
+The `obs` DataFrame is now augmented with the following new metadata, allowing users to forego common calculations used in early data pre-processing:
+
+- `raw_sum`: Represents the count sum derived from X["raw"]
+- `nnz`: Enumerates the number of non-zero (nnz) values
+- `raw_mean`: Provides the average count of nnz values
+- `raw_variance`: Indicates the count variance of nnz values
+- `n_measured_vars`: Enumerates the "measured" genes, determined by the sum of the presence matrix
+
+## How to Use the New Features
+
+### Exporting Normalized Data
+
+Normalized data can be exported into AnnData with the following code:
+
+```python
+import cellxgene_census
+
+with cellxgene_census.open_soma() as census:
```

Comment:
This code is not going to work because the latest version of the Census data is not being opened.
```suggestion
with cellxgene_census.open_soma(census_version = "latest") as census:
```